docs/architecture/accounts.md

```diff
@@ -1,211 +1,139 @@
-    export.basic::auth_tx_rpo_falcon512
-  ```
-</details>
+An `Account` is composed of several core components, illustrated below:
 
-[Note scripts](notes.md#the-note-script) or transaction scripts can call `receive_asset`, `create_note`, and `move_asset_to_note` procedures.
+<p style="text-align: center;">
+    <img src="../img/architecture/account/account-definition.png" style="width:30%;" alt="Account diagram"/>
+</p>
 
-Transaction scripts can also call `auth_tx_rpo_falcon512` to authenticate the transaction.
+These components are:
 
-> **Warning**
-> Without correct authentication, i.e., knowing the correct private key, a note cannot successfully invoke `receive_asset`, `create_note`, or `move_asset_to_note`.
+1. [ID](#id)  
+2. [Storage](#storage)  
+3. [Nonce](#nonce)  
+4. [Vault](#vault)  
+5. [Code](#code)
 
-##### Basic fungible faucet (faucet for fungible assets)
+### ID
 
-There is also a standard for a [basic fungible faucet](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/asm/miden/contracts/faucets/basic_fungible.masm).
+> An immutable and unique identifier for the `Account`.
 
-<details>
-  <summary>Fungible faucet code</summary>
+The `Account` ID is a single field element (`felt`) of about 63 bits. The four most significant bits encode:  
+- [**Account type:**](#the-accounts-type) basic or faucet.  
+- [**Account storage mode:**](#the-accounts-storage-mode) public or private.
 
-  ```arduino
-  #! Distributes freshly minted fungible assets to the provided recipient.
-  #!
-  #! ...
-  export.distribute
-      # get max supply of this faucet. We assume it is stored at pos 3 of slot 1
-      push.METADATA_SLOT exec.account::get_item drop drop drop
-      # => [max_supply, amount, tag, note_type, RECIPIENT, ...]
+This encoding allows the ID to convey both the account’s unique identity and its operational settings.
 
-      # get total issuance of this faucet so far and add amount to be minted
-      exec.faucet::get_total_issuance
-      # => [total_issuance, max_supply, amount, tag, note_type RECIPIENT, ...]
+### Storage
 
-      # compute maximum amount that can be minted, max_mint_amount = max_supply - total_issuance
-      sub
-      # => [max_supply - total_issuance, amount, tag, note_type, RECIPIENT, ...]
+> A flexible, arbitrary data store within the `Account`.
 
-      # check that amount <= max_supply - total_issuance, fails if otherwise
-      dup.1 gte assert.err=ERR_BASIC_FUNGIBLE_MAX_SUPPLY_OVERFLOW
-      # => [asset, tag, note_type, RECIPIENT, ...]
+The [storage](../../objects/src/accounts/storage/mod.rs) of an `Account` consists of up to 255 indexed [storage slots](../../objects/src/accounts/storage/slot/mod.rs). Each slot can be one of the following types:
 
-      # creating the asset
-      exec.asset::create_fungible_asset
-      # => [ASSET, tag, note_type, RECIPIENT, ...]
+- **`StorageSlot::Value`:** Contains a single `Word` (32 bytes) of arbitrary data.  
```

Comment:
I wonder if `Word` is a common term. Or if this is confusing. 

Is it clearer if we say: "Contains 32 bytes of arbitrary data"?